yaml_specs/modulemd_stream_v2.yaml

```diff
@@ -91,8 +126,6 @@ data:
     # the runtime block so that it matches the stream the module was built
     # against.  Multiple builds result in multiple output modulemd files.
     # See below for an example.
-    # TODO: Provides, conflicts, obsoletes, recommends, etc.
-    #       Do we even need those?
```

Comment:
As said above, this will likely become necessary soon. Not sure why we don't have it yet, actually...

Reply:
I removed these from the specification because I didn't think *proposed* concepts belong in the spec (it confuses the reader). Those should be filed as tickets in the issue tracker instead.

Reply:
If you move them to the issue tracker, then this will look good to merge.

Reply:
https://github.com/fedora-modularity/libmodulemd/issues/448